index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Software Developer Training School</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body class="homepage">
+    <section class="intro">
+        <div class="intro-sect">
+            <header class="nav-bar">
```

Comment:
You did a great job adding sections to your project! Please notice that, in this case, your header should be outside the `intro` section. This will make your project semantically accurate and improve accessibility 😄.
[Check this if you want to learn more.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)